src/functions/Mock.ps1

```diff
@@ -28,20 +28,22 @@ function New-MockBehavior {
         [Parameter(Mandatory)]
         $Hook,
         [string[]]$RemoveParameterType,
-        [string[]]$RemoveParameterValidation
+        [string[]]$RemoveParameterValidation,
+        [switch]$AllowFallback
     )
 
     [PSCustomObject] @{
-        CommandName = $ContextInfo.Command.Name
-        ModuleName  = $ContextInfo.TargetModule
-        Filter      = $ParameterFilter
-        IsDefault   = $null -eq $ParameterFilter
-        IsInModule  = -not [string]::IsNullOrEmpty($ContextInfo.TargetModule)
-        Verifiable  = $Verifiable
-        Executed    = $false
-        ScriptBlock = $MockWith
-        Hook        = $Hook
-        PSTypeName  = 'MockBehavior'
+        CommandName   = $ContextInfo.Command.Name
+        ModuleName    = $ContextInfo.TargetModule
+        Filter        = $ParameterFilter
+        AllowFallback = $AllowFallback
```

Comment:
Name suggestions for parameter and property? Fallback is also used for "fallback to default mock". `AllowOriginal` could give expectation of blocking stuff like `& (Get-Command Get-ChildItem -CommandType Cmdlet)`. `AllowPassthrough` is unclear. Other short suggestions?

Focus on other review comments first in case this design is discarded.